packages/lib/domainManager/deploymentServices/vercel.ts

```diff
@@ -93,7 +93,7 @@ function handleDomainCreationError(error: { code?: string | null; domain?: strin
     return true;
   }
 
-  const errorMessage = `Failed to take action for domain: ${error.domain}`;
+  const errorMessage = `Failed to create domain on Vercel: ${error.domain}`;
```

Comment:
Improved the error message in the PR as I saw the error while testing and it wasn't clear enough